change/@internal-storybook-dc3f30cf-69d5-4778-9f40-fb10fae10863.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Should this be a patch change?

Reply:
maybe we keep prelease as an option?

Reply:
My comment was more about how to label storybook changes. They don't impact the library at all. So... none? Or patch?

Reply:
if no impact, I would say "none"